nuxt/components/form/Form.vue

```diff
@@ -61,6 +68,26 @@ const props = withDefaults(defineProps<Props>(), {
   submitName: undefined,
 })
 
+const store = useMaevsiStore()
+const turnstileKey = ref('')
+const disabledSubmitButton = ref(true)
+
+watch(turnstileKey, (newKey) => {
+  if (newKey !== undefined || newKey !== '') {
+    disabledSubmitButton.value = false
+  }
+})
+
+function setTurnstileKeyAndEmit() {
```

Comment:
Let's use the modern variable declaration for new functions

```suggestion
// methods
const setTurnstileKeyAndEmit = () => {
```